### PR TITLE
XW-86 | Do not init session for WikiaMobile for anons

### PR DESCRIPTION
**Pull request by vforge**
https://wikia-inc.atlassian.net/browse/XW-86

@hakubo

### CODE REVIEW

File: extensions/wikia/UserLogin/UserLoginHelper.class.php [hunk 1]
```diff
@@ -431,6 +431,15 @@ public function addNewUserLogEntry( $user, $byEmail = false ) {
 	}
 
 	/**
+	 * Read a login token, DO NOT init if it doesn't exist
+	 *
+	 * @return string loginToken|null
+	 */
+	public static function readLoginToken() {
+		return LoginForm::getLoginToken() ? LoginForm::getLoginToken() : null;
```
Comment (bognix) [inline extensions/wikia/UserLogin/UserLoginHelper.class.php]: you are calling this function twice

File: extensions/wikia/UserLogin/UserLoginHooksHelper.class.php [hunk 2]
```diff
@@ -123,8 +123,8 @@ public static function isValidEmailAddr( $addr ) {
 	public static function onMakeGlobalVariablesScript( Array &$vars ) {
 		$app = F::app();
 
-		if ( $app->checkSkin( 'wikiamobile' ) ) {
```
Comment (bognix) [inline extensions/wikia/UserLogin/UserLoginHooksHelper.class.php:126]: why did you remove this white space?

Comment (vforge) [inline extensions/wikia/UserLogin/UserLoginHooksHelper.class.php:126]: Mistake, I've edited this line and it got wrongly auto-formatted.

**Comment by bognix** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM
